indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -276,6 +276,8 @@
         kms-index (kf/get-kms-index context)
         platforms2-nested (map #(platform/platform2-nested-fields->elastic-doc kms-index %)
                                (map :short-name platforms))
+        ;; _ (Thread/sleep (rand-int 5000))
+        ;; _ (println "---- platforms2-nested" platforms2-nested)
```

Comment:
These prints aren't in my local copy.  Not sure how they're leaking into here...?